.test.sh

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env nix-shell
```

Comment:
Can you please add a Apache-2 license to this file? (Copy and paste from an existing file?)